taskmanager/taskeditor/forms.py

```diff
@@ -32,3 +32,19 @@ class NewUserForm(UserCreationForm):
     class Meta:
         model = User
         fields = ['username', 'first_name', 'last_name', 'email', 'password1', 'password2']
+
+
+class GroupForm(ModelForm):
+    class Meta:
+        model = Group
+        fields = ['group']
+        widgets = {
+            'group': TextInput(attrs={
+                'class': 'form-control',
+                'placeholder': 'Enter group name'
+            }),
+        }
+
+    def save(self, commit=True, user=None):
+        self.instance.user = user
+        return super(GroupForm, self).save(commit=commit)
```

Comment:
И выше. Может ещё где-то

```suggestion
        return super().save(commit=commit)
```